src/pages/get-supporter-edition.astro

```diff
@@ -120,10 +125,12 @@ const metadata = {
   />
 
   <FeaturesList
-    title="Full comparison"
+    title="Full comparison<sup class='text-xl'>*</sup>"
```

Comment:
GH won't let me comment on unchanged lines...

On line 430, is it time to remove the "coming soon"?

Reply:
Fixed.